deploy/olm-catalog/gitops-operator/metadata/annotations.yaml

```diff
@@ -5,3 +5,6 @@ annotations:
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: gitops-operator
+  operators.operatorframework.io.metrics.builder: operator-sdk-v0.19.0
```

Comment:
Do we need these changes ?

Reply:
These were added automatically after running the script to build operator-index, wasn't sure if they're important so left them in
I can remove them though